src/pykeen/contrib/lightning.py

```diff
@@ -268,6 +273,26 @@ def _dataloader(
 )
 
 
+def lit_pipeline(
+    training_loop: HintOrType[LitModule] = None,
+    training_loop_kwargs: OptionalKwargs = None,
+    trainer_kwargs: OptionalKwargs = None,
+) -> None:
+    """
+    Create a :class:`LitModule` and run :class:`pytorch_lightning.Trainer` with it.
+
+    :param training_loop:
+        the training loop or a hint thereof
+    :param training_loop_kwargs:
+        keyword-based parameters passed to the respective :class:`LitModule` subclass upon instantiation.
+    :param trainer_kwargs:
+        keyword-based parameters passed to :class:`pytorch_lightning.Trainer`
+    """
+    lit = lit_module_resolver.make(training_loop, pos_kwargs=training_loop_kwargs)
+    trainer = pytorch_lightning.Trainer(**(trainer_kwargs or {}))
+    trainer.fit(model=lit)
```

Comment:
what's the point of this? it doesn't return anything

Reply:
It modifies the model's weights in-place

Reply:
maybe add a bit of extra description then? for poor fools like me ;)